### PR TITLE
Add Firefox 122 in Quay Test Image

### PR DESCRIPTION
**Pull request by LiZhang19817**
Add Firefox 122 in Quay Test Image

### CODE REVIEW

**Comment by LiZhang19817** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-aro-ocp-quay-e2e-tests-quay310-aro-ocp415

File: ci-operator/config/quay/quay-tests/quay-quay-tests-master.yaml [hunk 1]
```diff
@@ -18,6 +18,11 @@ images:
         yum install -y yum-utils && yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo && \
         yum -y install terraform \
         yum clean all && rm -rf /var/cache/yum/*
+    RUN curl -s "https://download-installer.cdn.mozilla.net/pub/firefox/releases/122.0/linux-x86_64/en-US/firefox-122.0.tar.bz2" -o "firefox-122.0.tar.bz2" && \
```
Comment (dongboyan77) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-master.yaml:21]: Suggest to use below url, it won't need to update per latest version
`curl -Lo firefox.tar.bz2 'https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US'`

Comment (LiZhang19817) [reply ci-operator/config/quay/quay-tests/quay-quay-tests-master.yaml]: Update.

**Pull request merged**